pkg/assets/assets_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// nolint:testpackage
```

Comment:
(I already wrote this comment, but either GitHub hides it from me, or I didn't add it to the review)

with this tests, we test only public API, so it's fine to use `assets_test` IMO.